github/VisualStudio
# Use custom signed System.Reactive.

Use a custom-signed version of `System.Reactive 4.0.0` built from the `ghfvs` branch at https://github.com/editor-tools/reactive.

This prevents the problems outlined in #1775.

Testing:

- Open the `Developer Command Prompt for Visual Studio 2017`
- Go into the `build\Debug` or `build\Release` directory
- Run `sn -T System.Reactive.dll`
- You should see a different public key token before building this branch to after

Depends on #1970
